Makefile

```diff
@@ -12,8 +12,11 @@ lint:
 vet:
 	go vet ./...
 
-fmt:
-	! gofmt -d -e -s *.go **/*.go 2>&1 | tee /dev/tty | read
+fmt-check:
+	@if test -n $(gofmt -l .); then \
```

Comment:
`gofmt -l .` seems to also be fmt-ing the `vendor` dir too.
We should find a way to change that ( maybe `grep -v "vendor/" ? ).

Reply:
Good catch! On for update...